script.js

```diff
@@ -115,11 +115,14 @@ document.addEventListener('DOMContentLoaded', function() {
     var requestStatusSelect = document.querySelector('#request-status-select');
     var requestOrganizationSelect = document.querySelector('#request-organization-select');
 
-    window.location.search = $.param({
-      query: quickSearch && quickSearch.value,
-      status: requestStatusSelect && requestStatusSelect.value,
-      organization_id: requestOrganizationSelect && requestOrganizationSelect.value
-    });
+    window.location.search = String.prototype.concat(
+      'query=',
+      quickSearch && quickSearch.value,
+      'status=',
+      requestStatusSelect && requestStatusSelect.value,
+      'organization_id=',
+      requestOrganizationSelect && requestOrganizationSelect.value
+    );
```

Comment:
Does this solution work when multiple params are selected? Do they not need to joined with `&`?

A couple more questions:
- Do we need to encode any of these values?
- Could `requestOrganizationSelect.value` or `requestStatusSelect.value` be `null` or `undefined`?